### ISSUE TITLE
Missing `<main>` landmark on Experian personal details page

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4078]
## Area or Component
Sign up form - Experian personal details

## Quick Summary
There's no `<main>` element on the page, so none of the elements are contained by landmark.s

## Accessibility Violation
All page content should be contained by landmarks: https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome
Document should have one main landmark: https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome

## Action Requested
Add a `<main>` landmark to the personal details form

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=94:109

**Comment by emmastephenson** [comment]
Fixed by #4276
